docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/slurm-requirements.rst

```diff
@@ -181,6 +181,43 @@ to optimize how Determined interacts with PBS:
    define ``CUDA_VISIBLE_DEVICES`` appropriately or provide the ``pbs.slots_per_node`` setting in
    your experiment configuration to indicate how many GPU slots are intended for Determined to use.
 
+-  Configure PBS to report GPU Accelerator type.
+
+   PBS administrators should set the value for ``resources_available.accel_type`` on each node with
```

Comment:
'should' seems too strong as this is not a requirement is it?

Reply:
Rephrased text and updated image in the PR description.